src/test/java/com/google/api/codegen/testdata/csharp_gapic_resourcenames_library.baseline

```diff
@@ -780,16 +780,16 @@ namespace Google.Example.Library.V1
     public partial class FindRelatedBooksRequest
     {
         /// <summary>
-        /// <see cref="IList{BookName}"/>-typed view over the <see cref="Names"/> resource name property.
+        /// <see cref="ResourceNameList{BookName}"/>-typed view over the <see cref="Names"/> resource name property.
         /// </summary>
-        public IList<BookName> BookNames =>
+        public ResourceNameList<BookName> BookNames =>
```

Comment:
Nope, I'm still confused - this is a read-only, computed property... what do we need to initialize? (Can you refer to a file/line number where this is being used?)